tensorflow/compiler/mlir/tosa/transforms/passes.h

```diff
@@ -17,18 +17,19 @@ limitations under the License.
 #define TENSORFLOW_COMPILER_MLIR_TOSA_TRANSFORMS_PASSES_H
 
 #include <memory>
+#include <unordered_set>
 
 #include "mlir/Pass/Pass.h"  // from @llvm-project
 
 namespace mlir {
 namespace tosa {
 
 void populateLegalizeTFPatterns(MLIRContext* ctx, RewritePatternSet& patterns);
-void populateLegalizeTFLPatterns(MLIRContext* ctx, RewritePatternSet& patterns);
+void populateLegalizeTFLPatterns(MLIRContext* ctx, RewritePatternSet& patterns, std::unordered_set<std::string> legalization_disable = {});
 
 std::unique_ptr<OperationPass<FuncOp>> createLegalizeTFPass();
 std::unique_ptr<OperationPass<FuncOp>> createFuseBiasTFPass();
-std::unique_ptr<OperationPass<FuncOp>> createLegalizeTFLPass();
+std::unique_ptr<OperationPass<FuncOp>> createLegalizeTFLPass(std::unordered_set<std::string> legalization_disable = {});
```

Comment:
Can you document all these APIs? In particular when they are taking options it is fairly important to carefully document these.

Reply:
I am assuming by document you mean C++ code comments in the passes.h file ?

Reply:
Right, I meant document at least the pass your changing, the others are "bonus" :)